tests/snuba/api/endpoints/test_discover_saved_queries.py

```diff
@@ -319,7 +319,7 @@ def test_post_cannot_use_version_two_fields(self):
                     "range": "24h",
                     "limit": 20,
                     "environment": ["dev"],
-                    "yAxis": "count(id)",
+                    "yAxis": ["count(id)"],
```

Comment:
@edwardgou-sentry Saved discover queries might need to be migrated to use this updated schema.

Reply:
Hey Alberto, I _think_ we may be able to get away with not having to do a migration for this change. Right now, queries are saved as JSON string in psql, ie:
```{"environment":[],"fields":["title","event.type","project","user.display","timestamp","failure_count()","avg(transaction.duration)"],"range":"24h","orderby":"-timestamp","widths":[],"yAxis":"count()","display":"default"}```

This should allow us to just start pushing yAxis values as Arrays without having to do any psql schema changes. I've also tested this on local to check for backwards compatibility. Let me know if I might be missing any steps and if this sounds right to you, thanks!